CMakeLists.txt

```diff
@@ -559,8 +559,6 @@ enable_clang_tidy(
         -bugprone-macro-parentheses
         # too many narrowing conversions in our code
         -bugprone-narrowing-conversions
-        -cert-dcl37-c
```

Comment:
@averinevg  Can we move all these to the `.clang-tidy` file ?

Reply:
Done